nautobot/ipam/tables.py

```diff
@@ -453,6 +453,10 @@ class IPAddressTable(StatusTableMixin, RoleTableMixin, BaseTable):
     )
     tenant = TenantColumn()
     parent__namespace = tables.Column(linkify=True)
+    interface_count = tables.Column(verbose_name="Interfaces")
+    interface_parent_count = tables.Column(verbose_name="Devices")
+    vm_interface_count = tables.Column(verbose_name="VM Interfaces")
+    vm_interface_parent_count = tables.Column(verbose_name="Virtual Machines")
```

Comment:
Looking at the screenshot, I wonder if it would be a nicer use of screen space to consolidate any of these columns via a TemplateColumn or two, either something like:

| Devices (Interfaces) | Virtual Machines (Interfaces) |
|-|-|
| 5 (10) | 1 (2) |

or 

| Devices / VMs | Device / VM Interfaces |
|-|-|
| 5 / 1 | 10 / 2 |

I guess then we'd miss out on sortability though, hmm?

Reply:
These columns aren't showing up by default now so you can pick which columns you care about. The sortability is probably important?